src/main.js

```diff
@@ -5,9 +5,9 @@ import ScholarsCV from './ScholarsCV'
 import 'jquery'
 import 'babel-polyfill'
 
-require ('bootstrap')
+//require ('bootstrap')
```

Comment:
should probably remove bootstrap dependency, and all css styles etc.. from code at some point